keras_cv/models/stable_diffusion/stable_diffusion.py

```diff
@@ -228,6 +233,216 @@ def generate_image(
         decoded = ((decoded + 1) / 2) * 255
         return np.clip(decoded, 0, 255).astype("uint8")
 
+    def text_to_image_prompt_to_prompt(
+        self,
+        prompt: str,
+        prompt_edit: str,
+        method: str,
+        self_attn_steps: Union[float, Tuple[float, float]],
+        cross_attn_steps: Union[float, Tuple[float, float]],
+        attn_edit_weights: np.ndarray = np.array([]),
+        negative_prompt: Optional[str] = None,
+        num_steps: int = 50,
+        unconditional_guidance_scale: float = 7.5,
+        batch_size: int = 1,
+        seed: Optional[int] = None,
+    ) -> np.ndarray:
+        """Generate an image based on the Prompt-to-Prompt editing method.
+        Edit a generated image controlled only through text.
+
+        Reference:
+
+        - "Prompt-to-Prompt Image Editing with Cross-Attention Control."
+        Amir Hertz, Ron Mokady, Jay Tenenbaum, Kfir Aberman, Yael Pritch, Daniel Cohen-Or.
+        https://arxiv.org/abs/2208.01626
+
+        Args:
+            prompt (str): Text containing the information for the model to generate.
+            prompt_edit (str): Second prompt used to control the edit of the generated image.
+            method (str): Prompt-to-Prompt method to chose. Can be ['refine', 'replace', 'reweight'].
```

Comment:
I think some documentation here (or a pointer to the examples) about the meaning of refine, replace, and reweight would be good

Reply:
Fixed @ianstenbit! Can you take a look and give some feedback on it?